ubuntu/install_gtest/action.yml

```
@@ -0,0 +1,20 @@
+name: 'install_gtest'
+description: 'Install and setup GTest for linking and building test application in Linux'
+
+runs:
+  using: composite
+  steps:
+
+    - name: install_apt_packages
+      run: |
+        echo "::group::Install GTest and GMock"
+        sudo apt install libgtest-dev libgmock-dev
```

Let's try to keep same format in every action
```suggestion
      run: |
      
        echo "::group::Install GTest and GMock"
        
        sudo apt install libgtest-dev libgmock-dev
```